pkg/interface/src/App.js

```diff
@@ -4,6 +4,8 @@ import * as React from 'react';
 import { BrowserRouter as Router, Route, withRouter, Switch } from 'react-router-dom';
 import styled, { ThemeProvider, createGlobalStyle } from 'styled-components';
 
```

Comment:
I think we should break this file into an outer wrapper file that handles all the logic, and then have the Router with routes in a separate file (with the router being a child component of the wrapper file). Starting to get big enough that that makes sense.